packages/admin-panel/src/importExport/ImportModal.js

```diff
@@ -3,39 +3,67 @@
  * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
  */
 
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { importData } from './actions';
 import { ImportExportModal } from './ImportExportModal';
 
-export const ImportModalComponent = ({ onImport, isOpen, ...props }) => {
+export const ImportModalComponent = ({ onImport, isOpen, errorMessage, ...props }) => {
   const [file, setFile] = useState(null);
+  const [value, setValue] = useState('');
+
+  // Handle case of the file changing since it was uploaded
+  useEffect(() => {
+    if (errorMessage === 'Failed to fetch') {
+      setFile(null);
+      setValue('');
+    }
+  }, [errorMessage]);
+
+  // Clear the file field state when the modal opens and closes
+  useEffect(() => {
+    setFile(null);
+    setValue('');
+  }, [isOpen]);
 
   const handleFiles = ({ target }) => {
     setFile(target.files[0]);
+    setValue(target.value);
   };
 
+  const fileErrorMessage =
+    errorMessage === 'Failed to fetch'
```

Comment:
I agree that this is a bit fragile - there are certainly other reasons that this error message would be returned. Are we able to detect anything more specific about the error response? Even if that's in `TupaiaApi.js` and we set a more specific error message over there?

Reply:
hmmm this is turning out to be a very tricky bug. The request with the changed file actually fails on the browser and doesn't even reach there server which rules out a server side fix. And from the browser it simply returns an error object  `{error: 'Failed to fetch'}`. There is an open issue for this in chromium as a few others have had a similar issue but that is un-resolved. I think our only options are to leave it as is or to go with the less than ideal workaround in this PR. What do you think?

Reply:
true, bummer

let's go with this workaround, but with a couple of tweaks:
- add a comment with a link to a tech debt card, in turn with a link to the open chromium issue
- change the error message to allow ambiguity, in case it 'failed to fetch' for another reason...something like 'Failed to upload, probably because the import file has been edited. Please reselect it and try again.'